larpandoradlcontent/LArDeepLearning/DeepLearningTrackShowerIdAlgorithm.h

```diff
@@ -0,0 +1,50 @@
+/**
+ *  @file   larpandoradlcontent/LArMonitoring/DeepLearningTrackShowerIdAlgorithm.h
+ *
+ *  @brief  Header file for the deep learning track shower id algorithm.
+ *
+ *  $Log: $
+ */
+#ifndef LAR_DEEP_LEARNING_TRACK_SHOWER_ID_ALGORITHM_H
+#define LAR_DEEP_LEARNING_TRACK_SHOWER_ID_ALGORITHM_H 1
+
+#include "Pandora/Algorithm.h"
+
+namespace lar_content
```

Comment:
Or should this be in lar_dl_content?

Reply:
My preference here was to keep the single namespace, as even with the additional library, it's all still LAr content. Can change this if you prefer, however.